sh_scrapy/extension.py

```diff
@@ -2,7 +2,7 @@
 from weakref import WeakKeyDictionary
 from scrapy import signals, log
 from scrapy.exceptions import NotConfigured
-from scrapy.contrib.exporter import PythonItemExporter
+from scrapy.exporters import PythonItemExporter
```

Comment:
As we're using `Scrapy=1.0.5` in `hworker` stack, I guess it's fine to update the path?